test/core/utils/preload-media.js

```diff
@@ -0,0 +1,76 @@
+/* global Promise */
+
+describe('axe.utils.preloadMedia', function() {
+	'use strict';
+
+	var origFn = axe.utils.preloadMedia;
+	var fixture = document.getElementById('fixture');
+	var fixtureSetup = axe.testUtils.fixtureSetup;
+
+	function rebuildTree() {
```

Comment:
Please take this out. You can just as easily put that line in the tests, and have a much more readable test because of it. Having readable tests is more important then having DRY tests.